okonomiyaki/runtimes/runtime.py

```diff
@@ -98,7 +98,7 @@ def from_prefix_and_platform(cls, prefix, platform, version=None,
             scriptsdir = bindir = posixpath.join(prefix, "bin")
             paths = (bindir, )
 
-        if version.major == 3:
+        if version.major == 3 and platform.os_kind != OSKind.windows:
```

Comment:
For sure, I know that `python3` is wrong for windows. But I do not have an opinion for posix systems. @sjagoe might have more to say about this